framework/compose/kodein-di-framework-compose/src/commonMain/kotlin/org.kodein.di.compose/CompositionLocal.kt

```diff
@@ -14,7 +15,7 @@ import org.kodein.di.DI
  *
  * @throws [IllegalStateException] if no DI container is attached to the Composable tree
  */
-internal val LocalDI: ProvidableCompositionLocal<DI?> = compositionLocalOf { null }
+public val LocalDI: ProvidableCompositionLocal<DI?> = staticCompositionLocalOf { null }
```

Comment:
This is `staticCompositionLocal` as, while possible, a DI instance is very unlikely to change during composition after it has been initially provided.